docs/source/specification.rst

```diff
@@ -0,0 +1,30 @@
+.. _specification:
+
+====================================================
+The Reproducible Execution Environment Specification
+====================================================
+
+repo2docker scans a repository for particular :ref:`config-files`, such
+as ``requirements.txt`` or ``REQUIRE``. The collection of files, their contents,
+and the resulting actions that repo2docker takes is known
+as the **Reproducible Execution Environment Specification** (or REES).
+
+The goal of the REES is to automate and encourage existing community best practices
+for reproducible computational environments. This includes installing
+community-standard specification files such as ``requirements.txt`` or ``REQUIRE`` using
+standard tools such as ``pip`` or ``conda`` or ``apt``. While repo2docker automates the
```

Comment:
I'd mention the files and corresponding tools instead of having `requirements.txt` which matches with `pip` but then `REQUIRE` which doesn't match with `conda` and `apt` which is all by itself.

Reply:
see latest push, how's that?

Reply:
Didn't seem to change these lines?

Reply:
oops, I forgot to save the file before pushing the latest commit :-)